var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -43,7 +44,9 @@ class Kaldi(Package):  # Does not use Autotools
     depends_on("cub", when="@2019-07-29:^cuda@:10")
 
     patch("openfst-1.4.1.patch", when="@2015-10-07")
-    patch("0001_CMakeLists_txt.patch", when="+cuda@11:")
+    patch("0001_CMakeLists_txt.patch", when="@:2021-11-16")
```

Comment:
Is there a reason `+cuda@11:` was removed from the `when` argument?

Reply:
> Is there a reason `+cuda@11:` was removed from the `when` argument?

Yes, simplicity. The patch doesn't break anything for non-cuda builds, just like the ones I added.